compiler/testData/codegen/bytecodeText/intRangeNoBoxing.kt

```diff
@@ -3,16 +3,9 @@ fun foo() {
     val range = 1 until 2
     for (i in range) {
     }
-
-    for (i in 1..2 step 4) {}
```

Comment:
Why it's removed?

Reply:
I removed it because the behavior is different now with JVM IR (no calls to getFirst/Last). But since now we have the JVM_IR_TEMPLATES I'll just change the file.

Reply:
Yes, please